GA-worksheet.md

```diff
@@ -25,45 +25,45 @@
 
 ##Project #2 Part #2: Style the Header with CSS
 
-1. What advantages does the `<link>` tag have over the `<style>` tag?
+1. What advantages does the `<link>` tag have over the `<style>` tag? Instead of putting your CSS directly in the html file and taking up space-like the <style> tag, the <link> tag, is a connection from your html to a css, that has all he properties and attributes you want. It's advantageous because it makes your html shorter and easier to read-something especially convenient when you have dozens of style attributes.
 
-2. What is normalize.css?
+2. What is normalize.css? it's a "thing" that basically renders the elements in your page consistent across all browsers and wherever your page is being seen, so it doesn't look messy.
 
-3. Why do we need to add a padding property to the `<ul>` tag?
+3. Why do we need to add a padding property to the `<ul>` tag? to clear an area aroud nd the/a element and center the elements better.
 
-4. What does the “inline” property do to a list of items?
+4. What does the “inline” property do to a list of items? it makes them appear next to each other in the same line by default.
 
 5. What do the following padding values do?
 
     padding: 25px 50px 75px 100px;
 
-    * top padding is __px
-    * right padding is __px
-    * bottom padding is __px
-    * left padding is __px
+    * top padding is 25px
+    * right padding is 50px
+    * bottom padding is 75px
+    * left padding is 100px
 
     padding: 20px;
 
-    * top padding is __px
-    * right padding is __px
-    * bottom padding is __px
-    * left padding is __px
+    * top padding is 20px
+    * right padding is 20px
+    * bottom padding is 20px
+    * left padding is 20px
 
-6. What is the difference between padding and margin?
+6. What is the difference between padding and margin? Padding creates space on the inside of an element while margin creates space on the outside.
 
-7. What did border-radius do to the square border around the image?
+7. What did border-radius do to the square border around the image? It rounded off the corners.
 
 ##Project #2 Part #3: Responsive Design and Javascript
 
-1. Why is responsive design becoming a more important aspect to consider when making a website compared to ten years ago?
+1. Why is responsive design becoming a more important aspect to consider when making a website compared to ten years ago? Responsive design is becoming more important because people today are lazy and impatient, and like things to be pretty. The less clicks it takes for me to get where I want to go, the better. It's easier and faster. Moreover, today there are hundreds of thousands of websites, and I guess the prettiest one wins, or something. You have to make your website stand out both visually and in terms of how efficient it is.
```

Comment:
There is actually a field called UX (User Experience Design)  which is concerned with everything you wrote here. As programmers, we not only have to worry about writing good, clean code. We also have to worry about how the user experiences it.

Responsive design is also important now because we didn't have to worry about how websites looked on smart phones, because they didn't exist. The first iPhone came out in 2007!